src/main/java/frc/team2412/robot/subsystem/TargetLocalizer.java

```diff
@@ -22,7 +24,8 @@ public TargetLocalizer(DrivebaseSubsystem drivebase, ShooterSubsystem shooter, S
     }
 
     public double getDistance() {
-        return hasTarget() ? shooterVisionSubsystem.getDistance() + shooterSubsystem.getDistanceBias() : 0;
+        return hasTarget() ? shooterVisionSubsystem.getDistance() + shooterSubsystem.getDistanceBias()
```

Comment:
Does it make sense to smooth this with a moving average or similar for distance to help reduce constantly changing the motor speed via loss of target (wrapping, distracting lighting, fetching balls under hangar, adverse lighting, etc) and interpolation?  Given it takes so much longer to spin up flywheel, we may want to err on the side of keeping the flywheel speed up / gracefully slowing instead of slamming distance down to zero.